composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "^2.3.27 || ~3.0",
```

Comment:
it is also more efficient in the resolution btw, because the 2.4 versions are never considered instead of being considered and then rejected by the conflict.